packages/chat-component/src/main.ts

```diff
@@ -3,9 +3,9 @@ import { LitElement, html, css } from 'lit';
 import { customElement, query, property } from 'lit/decorators.js';
 import { globalConfig, requestOptions } from './config/global-config.js';
 import { processText } from './utils/index.ts';
-import { EventSourceParserStream } from 'eventsource-parser/stream';
 import { unsafeHTML } from 'lit/directives/unsafe-html.js';
 import type { BotResponse, ChatMessage, ChatMessageText, Citation, RequestOptions } from './types';
+import { readStream } from './core/stream/index.ts';
```

Comment:
.ts imports should not be allowed (we must review tsconfig!)

Reply:
Let's discuss this. Because the only reason we're importing .ts is to keep static typing in the text processing and decouple from barrel file.  Removing ts imports may lead to regressions.

Reply:
We can keep barrel imports, that's what I have too in the backend :)
But `.ts` imports (instead of `.js`) will break ESM compatibility, and it's something the TS compiler reports as an error by default (don't ask me why they don't automatically process the extension...)